src/main/java/core/basesyntax/List.java

```diff
@@ -1,6 +1,7 @@
 package core.basesyntax;
 
 public interface List<T> {
+    int INITIAL_SIZE = 10;
```

Comment:
lets move this constant to out List implementation? Maybe other implementations will have different initial size....